gui/default/assets/lang/lang-en.json

```diff
@@ -450,6 +463,7 @@
    "When adding a new folder, keep in mind that the Folder ID is used to tie folders together between devices. They are case sensitive and must match exactly between all devices.": "When adding a new folder, keep in mind that the Folder ID is used to tie folders together between devices. They are case sensitive and must match exactly between all devices.",
    "Yes": "Yes",
    "Yesterday": "Yesterday",
+   "You can also copy and paste the content into the message manually.": "You can also copy and paste the content into the message manually.",
```

Comment:
I actually like to keep translations bundled together with their commit, so if it needs to be reverted later for some reason, the unused translations can easily be removed with it as well. This wouldn't be necessary if the Go translate script simply refreshed the whole `json` file on each run (which I'd personally prefer, because it'd allow the person to see whether any possible changes to the script haven't broken anything), but that's a completely different story…

Reply:
Good point about the revert.  I don't feel strongly about this, just wanted to give a hint about not forgetting such changes while the PR is still a moving target.  Fine with me, feel free to resolve this.